ncm-spma/src/test/perl/yum-cleanup-repos.t

```diff
@@ -82,13 +82,22 @@ In this case, the error must be reported
 =cut
 
 initialize_repos();
-chmod(0500, $REPO_DIR);
+if ($>) {
+  chmod(0500, $REPO_DIR);
+} else {
+  diag("Bad test for root user. Trying to set immutable bit.");
+  system("chattr +i $REPO_DIR/*repo");
+}
 is($cmp->cleanup_old_repos($REPO_DIR, $repos), 0,
```

Comment:
yes, there is. `mvn clean` will fail. the alternative here would be to not run this test when running the tests as root. (would adding a message how to clean up be sufficient?)